src/backend/Modules/Account/Timebox.Account.Application/Services/AccountService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Threading.Tasks;
+using Timebox.Account.Application.DTOs;
+using Timebox.Account.Application.Interfaces.Repositories;
+using Timebox.Account.Application.Interfaces.Services;
+using Timebox.Account.Domain.Entities;
+
+namespace Timebox.Account.Application.Services
+{
+    public class AccountService : IAccountService
+    {
+        private readonly IAccountRepository _accountRepository;
+        private readonly IPasswordService _passwordService;
+        private readonly IEmailService _emailService;
+        private readonly IPhoneService _phoneService;
+
+        public AccountService(IAccountRepository accountRepository, IPasswordService passwordService, IEmailService emailService, IPhoneService phoneService)
+        {
+            _accountRepository = accountRepository;
+            _passwordService = passwordService;
+            _emailService = emailService;
+            _phoneService = phoneService;
+        }
+        
+        public async Task<AccountEntity> CreateAccountAsync(CreateAccountDTO createAccountDto)
+        {
+            if (!_emailService.IsValidEmailAddress(createAccountDto.Email))
+                return null; //ToDo: Throw
```

Comment:
We can merge these and I have done a base exception class and an example handler so we can both merge to master soon and get this shared code